src/ca.c

```diff
@@ -2977,3 +2978,34 @@ void set_ca_multiple_pmt(char *o) {
         }
     }
 }
+
+char *get_ca_caids_string(int i, char *dest, int max_len) {
+    if (!ca_devices[i])
+        ca_devices[i] = alloc_ca_device();
+    if (!ca_devices[i])
+        return NULL;
```

Comment:
@catalinii added the check here and dropped the earlier commit that removed these checks